src/lib/tokenizers/alpheios-remote-tokenizer.js

```diff
@@ -25,8 +25,11 @@ export default class AlpheiosRemoteTokenizer {
       }
     })
 
+    console.info('adapterTokenizerRes - ', adapterTokenizerRes)

```

Comment:
I would like to be able to distinguish 400 errors from 500 errors here.

400 errors from the tokenization service are those that are invalid input, and most likely (assuming no bugs) to be problems with the user input.

500 errors are system errors and most likely to be either bugs or service failures.

I think they should be differentiated for the user somehow.